examples/2-complex-example/src/modules/user/user.model.ts

```diff
@@ -31,4 +37,7 @@ export class User extends BaseModel {
   @Authorized('user:admin')
   @StringField({ nullable: true })
   privateField?: string;
+
+  @JSONField({ nullable: true })
+  jsonField?: any;
```

Comment:
Instead of using `any`, does it make sense to add an explicit `JSON` type?

Reply:
fixed